### ISSUE TITLE
NoMethodError on load

### ISSUE THREAD

**Issue by jonnyfiveiq** [DieboldInc/vcloud-ruby#1]
Hi, was excited to see this gem, and looking fwd to using it. But stumbled at the require 'vcloud' line, I get the following...Thanks

NoMethodError: private method `class_variable_set' called for VCloud::Catalog:Class
    from ./vcloud/base_vcloud_entity.rb:7:in`inherited'
    from /Users/johnhardy/Downloads/DieboldInc-vcloud-ruby-209ed70/lib/vcloud/user/catalog.rb:3
    from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:36:in `gem_original_require'
    from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:36:in`require'
    from ./vcloud/user.rb:3
    from ./vcloud/user.rb:2:in `each'
    from ./vcloud/user.rb:2
    from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:36:in`gem_original_require'
    from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:36:in `require'
    from ./vcloud.rb:14
    from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:36:in`gem_original_require'
    from /Library/Ruby/Site/1.8/rubygems/custom_require.rb:36:in `require'
    from (irb):2

**Comment by BrianMcClainDiebold** [comment]
Looking at you stacktrace, looks like you're using the OS X version of ruby. I tried using my built-in version (ruby 1.8.7 (2012-02-08 patchlevel 358) [universal-darwin12.0]), and was able to reproduce the error.

Unfortunately, we built this against 1.9.2 and 1.9.3, and I did test against 1.9.1 and it worked. Looks like that method was changed between 1.8 and 1.9

Both myself and @zrob are looking at what it would take to support 1.8.7 or older now

**Comment by jonnyfiveiq** [comment]
Thanks for your prompt response,   I updated to 1.9.3 and works straight
away. Good spot!

Now to get to work with itŠthanks

From:  Brian McClain notifications@github.com
Reply-To:  DieboldInc/vcloud-ruby
<reply+i-7795615-21f600bc29893fd879f1ee559e1c7fbcf8c72083-2084157@reply.gith
ub.com>
Date:  Tuesday, 23 October 2012 14:59
To:  DieboldInc/vcloud-ruby vcloud-ruby@noreply.github.com
Cc:  John Hardy john@jhardy.co.uk
Subject:  Re: [vcloud-ruby] NoMethodError on load (#1)

Looking at you stacktrace, looks like you're using the OS X version of ruby.
I tried using my built-in version (ruby 1.8.7 (2012-02-08 patchlevel 358)
[universal-darwin12.0]), and was able to reproduce the error.

Unfortunately, we built this against 1.9.2 and 1.9.3, and I did test against
1.9.1 and it worked. Looks like that method was changed between 1.8 and 1.9

Both myself and @zrob https://github.com/zrob  are looking at what it
would take to support 1.8.7 or older now

 ‹
 Reply to this email directly or view it on GitHub
https://github.com/DieboldInc/vcloud-ruby/issues/1#issuecomment-9702754 .
